attestation-agent/kbs_protocol/test/policy.rego

```diff
@@ -5,3 +5,7 @@ default allow = false
 allow {
 	input["tee"] == "sample"
 }
+
+allow {
+        input["tee"] == "se"
+}
```

Comment:
I think this policy file is intended to be used with the nontee (sample tests) so I wouldn't add the SE clause here. Isn't the fake SE attester always returning false for platform detection anyway?

Reply:
Yeah, this is what I added when testing se driver, we need drop it when merge.

Reply:
Fixed.